spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -136,4 +137,40 @@
       expect(user.notifications.count).to eq(1)
     end
   end
+
+  describe "#send_feed_customization_notification" do
+    let!(:user) { create(:user, :with_identity, identities: %w[twitter github], created_at: 3.days.ago) }
+
+    it "does not send a notification to a newly-created user" do
+      user.update!(created_at: Time.zone.now)
+      sidekiq_perform_enqueued_jobs { described_class.new(user.id).send(:send_feed_customization_notification) }
+      expect(Notification).not_to have_received(:send_welcome_notification)
+    end
+
+    it "does not send a notification to a user that is following 2 tags" do
+      2.times { user.follow(create(:tag)) }
+      sidekiq_perform_enqueued_jobs { described_class.new(user.id).send(:send_feed_customization_notification) }
+      expect(Notification).not_to have_received(:send_welcome_notification)
+    end
+
+    it "does send a notification to a user with 0 tag follows created 3 days ago" do
```

Comment:
```suggestion
    it "sends a notification to a user with 0 tag follows" do
```

does the created at 3 days ago really make a difference here?